### PR TITLE
add custom buttons option

### PR DESCRIPTION
**Pull request by tmburnell**
link to https://github.com/jpilfold/ngx-image-viewer/issues/4

Can you please review the var names and see if they make since to you.  Or would like me to change anything.

### CODE REVIEW

**Comment by tmburnell** [comment]
Do you mind if i add this change in with my request:
to change the cursor on disabled buttons.

```css
.img-container button {
    z-index: 99;
    position: absolute;
    right: 15px;
    cursor: pointer;
}
```
To
```css
.img-container button {
    z-index: 99;
    position: absolute;
    right: 15px;

    &:not(:disabled) {
      cursor: pointer;
    }
}
```

File: src/app/image-viewer/image-viewer.component.ts [hunk 1]
```diff
@@ -119,6 +121,10 @@ export class ImageViewerComponent implements OnInit {
     }
   }
 
+  fireCustomeEvent(name, index) {
```
[the remark this answers is not in the document]
Comment (tmburnell) [reply src/app/image-viewer/image-viewer.component.ts]: Will have this ready in a min.

**Comment by jpilfold** [comment]
@tmburnell How's this coming along?

**Comment by tmburnell** [comment]
@jpilfold Updated lint, function, naming.  Can you please review again.

**Comment by tmburnell** [comment]
Do you want a type definition for the event format?

**Comment by jpilfold** [comment]
Yes please, that would be good.

**Comment by tmburnell** [comment]
Have added the event ts type.

**Pull request merged**